open_schools_platform/common/admin.py

```diff
@@ -29,17 +34,30 @@ def choices(self, changelist):
         yield all_choice
 
 
-def admin_wrapper(admin_model: Type[BaseAdmin]):
-    admin_model.highlight_deleted_field.short_description = admin_model.field_to_highlight  # type: ignore[attr-defined]
+def hard_delete_queryset(self, request, queryset):
+    queryset.delete(force_policy=HARD_DELETE)
+
+
+def admin_wrapper(model: Type[BaseModel]):
+    def wrap_admin_model(admin_model: Type[BaseAdmin]):
+        delete_policy = model._safedelete_policy
+        if delete_policy:
+            if delete_policy == SOFT_DELETE or delete_policy == SOFT_DELETE_CASCADE:
+                admin_model.actions = ('undelete_selected',)
+                admin_model.highlight_deleted_field.short_description = admin_model.field_to_highlight  # type: ignore[attr-defined] # noqa
+
+                if admin_model.list_display:
+                    admin_model.list_display = BaseAdmin.list_display + admin_model.list_display  # type: ignore[operator] # noqa
```

Comment:
Can you please explain why we need "+" operator here? Are we merging list_displays this way?

Reply:
This is for concatenating the values of the field that were written in the base class